community-membership.md

```diff
@@ -236,12 +202,15 @@ The following apply to the subproject for which one would be an owner.
 
 The Maintainer role has been removed and replaced with a greater focus on [OWNERS].
 
-[code reviews]: contributors/devel/collab.md
-[community expectations]: contributors/guide/community-expectations.md
-[developer guide]: contributors/devel/README.md
-[two-factor authentication]: https://help.github.com/articles/about-two-factor-authentication
+[code reviews]: /contributors/devel/collab.md
+[community expectations]: /contributors/guide/community-expectations.md
+[contributor guide]: /contributors/guide/README.md
+[Kubernetes GitHub organizations]: /github-management#actively-used-github-organizations
 [kubernetes-dev@googlegroups.com]: https://groups.google.com/forum/#!forum/kubernetes-dev
-[sigs.yaml]: sigs.yaml
-[New contributors]: https://github.com/kubernetes/community/blob/master/CONTRIBUTING.md
-[OWNERS]: contributors/guide/owners.md
 [kubernetes-sigs]: https://github.com/kubernetes-sigs
+[membership request]: https://github.com/kubernetes/org/issues/new?template=membership.md&title=REQUEST%3A%20New%20membership%20for%20%3Cyour-GH-handle%3E
```

Comment:
Ahh looks like you actually did include the link. Seems like maybe it belongs above instead of, or in addition, to here.